frontend/src/views/TE/Task.vue

```diff
@@ -4,15 +4,21 @@
       <v-progress-circular indeterminate></v-progress-circular>
     </div>
     <div v-else>
-      <div :style="{ display: 'flex', flexWrap: 'wrap', alignItems: 'center' }">
-        <h1 class="d-inline" :style="{ width: 'auto' }">
-          {{ $title }}
+      <div>
+        <h1>
+          Track Editing Task
         </h1>
-        <v-chip :style="getTaskStyle(task)">{{ task.status }}</v-chip>
+        <h1>{{ $route.params.taskId }} =</h1>
```

Comment:
I think this can be done with breaking the line. Having two `h1` on the page is not a good pattern.